doc/index.rst

```diff
@@ -8,8 +8,9 @@ It is developed as a part of the `Karoo Array Telescope <http://ska.ac.za/>`_
 (KAT) project and used at KAT for the monitoring and control of hardware
 devices.
 
-The protocol specification :download:`NRF-KAT7-6.0-IFCE-002-Rev5.pdf`
-is maintained as an internal memo. The latest version is `Rev5`.
+The protocol specification :download:`NRF-KAT7-6.0-IFCE-002-Rev5-1.pdf`
+is maintained as an internal memo. The latest version is `Rev5.1`.
+The specification source is hosted here:  https://github.com/ska-sa/katcp-guidelines
```

Comment:
Will sphinx automatically turn that URL into a link?

Reply:
Yes, plain text like this does become a hyperlink (at least on readthedocs).